sites/public/src/patterns/CardList.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import { Heading, Card } from "@bloom-housing/ui-seeds"
+import styles from "./CardList.module.scss"
+
+interface ContentCardProps {
+  title: string
+  description?: React.ReactNode
+  priority?: 1 | 4 | 2 | 3 | 5 | 6
+  children?: React.ReactNode
+}
+
+export const ContentCard = (props: ContentCardProps) => {
+  return (
+    <Card spacing="sm" className={styles["content-card"]}>
+      <Card.Section>
+        <Heading size="md" priority={props.priority ?? 4} className={styles["header"]}>
+          {props.title}
+        </Heading>
+        {props.description && <div className={styles.description}>{props.description}</div>}
+        {props.children && <div>{props.children}</div>}
+      </Card.Section>
+    </Card>
+  )
+}
+
+export interface CardContent {
+  title: string
+  description: React.ReactNode
+}
+
+interface CardListProps {
+  cardContent: CardContent[]
+  priority?: 1 | 4 | 2 | 3 | 5 | 6
```

Comment:
Same here. Also since this is used twice could it be an explicit type at the top of this file?

Reply:
(Same as above)